src/HomeWork5/HomeWork5/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace HomeWork5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        { 
+            var cages = new IAnimal[3];
```

Comment:
Using array initializers is a better practice, you do not need to specify indexes explicitly, it makes your code more manageable